frontend/src/app/globals.css

```diff
@@ -1,10 +1,9 @@
 :root {
```

Comment:
Hmmm a lot of the changes in this file are linters / formatters not agreeing. Can we minimize those and agree on a format?